tests/EdFi.Ods.WebApi.CompositeSpecFlowTests/CompositeResourcesCanBeDefinedThroughXmlFeature.CompositeIncludesUnflattenedReference.approved.txt

```diff
@@ -13,6 +13,25 @@ where comp_aaa.Id = :Id AND (comp_aaa.SchoolId IN (
                             SELECT fltr_aab.StudentUSI 
                             FROM EdFi.Ods.Entities.NHibernate.QueryModels.Views.auth_StudentUSIToEducationOrganizationId fltr_aab 
                             WHERE fltr_aab.SourceEducationOrganizationId IN (:LocalEducationAgencyId)))
+order by comp_aaa.EntryDate,
```

Comment:
@semalaiappan What is the change that resulted in this additional query being executed for a composite? Is this bleeding in from another set of changes where the approval test update was missed? I wouldn't expect to see it on this PR.